concrete/js/ckeditor/package.json

```diff
@@ -27,7 +27,7 @@
   ],
   "_resolved": "https://registry.npmjs.org/ckeditor4/-/ckeditor4-4.14.0.tgz",
   "_spec": "4.14.0",
-  "_where": "/Users/mesparza/Sites/cv9/build",
+  "_where": "/Users/andrewembler/Projects/concrete5/build",
```

Comment:
lol. wtf are we gonna do about this stupid crap?

Reply:
Can this file be gitignored? Doesn't seem like we're actually wanting to integrate with npm at this point but maybe ckeditor relies on this file being publicly accessible?

Reply:
Sure, I'd be willing to try that for sure.